samples/gauges/bullet-graph/type-reversed/Pages/BulletGraphTypeReversed.razor

```diff
@@ -0,0 +1,114 @@
+                     TargetValue="80"
+                     MinimumValue="0"
+                     MaximumValue="100"
+                     Interval="10"
+                     LabelInterval="10"
+                     LabelExtent="0.0">
+            @for (int i = 0; i < RangeInfo1.Count; i++)
+            {
+                GaugeRangeInfo info1 = RangeInfo1[i];
+                <LinearGraphRange StartValue=info1.StartValue
+                                  EndValue=info1.EndValue
+                                  InnerStartExtent=info1.InnerStartExtent
+                                  InnerEndExtent=info1.InnerEndExtent
+                                  OuterStartExtent=info1.OuterStartExtent
+                                  OuterEndExtent=info1.OuterEndExtent
+                                  Brush="@info1.Brush"
+                                  Outline="@info1.Outline">
+                </LinearGraphRange>
+            }
+        </BulletGraph>
+        <BulletGraph Height="120px" Width="100%"
+                     Orientation="LinearScaleOrientation.Horizontal"
+                     BackingBrush="transparent"
+                     BackingOutline="transparent"
+                     TransitionDuration="500"
+                     Value="30"
+                     TargetValue="20"
+                     MinimumValue="0"
+                     MaximumValue="100"
+                     Interval="10"
+                     LabelInterval="10"
+                     LabelExtent="0.0">
+            @for (int i = 0; i < RangeInfo2.Count; i++)
+            {
+                GaugeRangeInfo info2 = RangeInfo2[i];
+                <LinearGraphRange StartValue=info2.StartValue
+                                  EndValue=info2.EndValue
+                                  InnerStartExtent=info2.InnerStartExtent
+                                  InnerEndExtent=info2.InnerEndExtent
+                                  OuterStartExtent=info2.OuterStartExtent
+                                  OuterEndExtent=info2.OuterEndExtent
+                                  Brush="@info2.Brush"
+                                  Outline="@info2.Outline">
+                </LinearGraphRange>
+            }
+        </BulletGraph>
+    </div>
+</div>
+
+@code {
+
+    private System.Collections.ObjectModel.ObservableCollection<GaugeRangeInfo> RangeInfo1;
+    private System.Collections.ObjectModel.ObservableCollection<GaugeRangeInfo> RangeInfo2;    
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();        
+        BulletGraphModule.Register(InfragisticsBlazor);
+
+        RangeInfo1 = new System.Collections.ObjectModel.ObservableCollection<GaugeRangeInfo>();
+        RangeInfo2 = new System.Collections.ObjectModel.ObservableCollection<GaugeRangeInfo>();       
+
+        string[] colors1 = "#008000,#10B401,#45EC03,#2EFA2E".Split(',');
+        string[] colors2 = "#CF0000,#FF0000,#FD3939,#FA6363".Split(',');        
```

Comment:
this is correct/standard way of creating string arrays:
string[] colors2 = new string[4] { "#CF0000","#FF0000","#FD3939","#FA6363" };
or use List which do not require specifying size of data structure:
var colors2 = new List<string>() { "#CF0000","#FF0000","#FD3939","#FA6363" };